include/geos/geom/PrecisionModel.h

```diff
@@ -163,6 +163,8 @@ class GEOS_DLL PrecisionModel {
     /// destructor
     ~PrecisionModel(void);
```

Comment:
Pretty sure we can just un-declare the destructor instead of declaring the assignment operator here.